apps/admin-x-activitypub/src/components/Activities.tsx

```
@@ -53,6 +54,13 @@ const getExtendedDescription = (activity: GroupedActivity): JSX.Element | null =
                 className='ap-note-content mt-1 line-clamp-2 text-pretty text-grey-700'
             ></div>
         );
+    } else if (activity.type === ACTIVITY_TYPE.ANNOUNCE && !activity.object?.name && activity.object?.content) {
+        return (
+            <div
+                dangerouslySetInnerHTML={{__html: stripHtml(activity.object?.content || '')}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)